media/css/firefox/whatsnew/whatsnew-105-en.scss

```diff
@@ -0,0 +1,59 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/protocol/fonts';
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/components/notification-bar';
+@import '~@mozilla-protocol/core/protocol/css/components/section-heading';
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import 'includes/base';
+@import 'includes/dark-mode';
+
+// * -------------------------------------------------------------------------- */
+// Main content
+.wnp-content-main {
+    padding-top: $spacing-md;
+    text-align: center;
+}
+
+.wnp-main-title {
+    @include text-title-xl;
+}
+
+.wnp-main-img {
+    max-width: $content-md;
```

Comment:
Issue (non-blocking): looks like this is preventing the image from resizing on smaller viewports?